### PR TITLE
Ensure search and filter field list is clickable after selecting a data field

### PR DESCRIPTION
**Pull request by maksim-an-upplabs**
@squallstar @tonytlwu 
Fixed "List search" and "List filters" fields become unclickable for the second time click on it.
ref https://github.com/Fliplet/fliplet-studio/issues/4231
video: https://cl.ly/880abb2013ed

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -1224,6 +1227,18 @@ var DynamicLists = (function() {
 
         // Update source
         field.data('bs.tokenfield').$input.autocomplete({source: newSource});
+        _this.removeFocusFromTokenInput();
+      });
+
+      $('input.tokenfield').on('tokenfield:createtoken', function (event) {
+        var currentTokens = $(this).tokenfield('getTokens');
+        var isExist = _.some(currentTokens, function (item) {
```
Comment (tonytlwu) [inline js/interface.js]:
```suggestion
        var tokenExists = _.some(currentTokens, function (item) {
```

Comment (maksim-an-upplabs) [reply js/interface.js]: @tonytlwu I'm sorry, I need to change variable name?

Comment (tonytlwu) [reply js/interface.js]: @maksim-an-upplabs That's correct. Please change it.

Comment (tonytlwu) [reply js/interface.js]: I see you've changed it already 👍

**Pull request merged**